ocaml/src/alba_client_download.ml

```diff
@@ -172,6 +172,8 @@ let download_fragment
         >>= E.return)
      >>== fun (t_decompress, (maybe_decompressed : Lwt_bytes.t)) ->
 
+     fragment_cache # add namespace_id cache_key maybe_decompressed >>= fun () ->
```

Comment:
this is a bug fix for the regression introduced by https://github.com/openvstorage/alba/pull/166